src/components/Toggle/README.md

```diff
@@ -1,15 +1,58 @@
 Default
 
 ```js
-<Toggle id='toggle1'/>
+class ToggleExample extends React.Component {
+  constructor() {
+    super()
+    this.state = { checked: true }
+
+    this.onClick = this.onClick.bind(this)
+  }
+
+  onClick() {
```

Comment:
You could `onClick = () =>` to avoid the `this.onClick = this.onClick.bind(this)` line.

Reply:
There's no arrow function available :(

Reply:
Opa. Then no.